docs/basic-usage/middleware.md

```diff
@@ -13,9 +13,18 @@ Route::group(['middleware' => ['can:publish articles']], function () {
 });
 ```
 
+In Laravel v10.9 and up, you can also call this middleware with a static method.
```

Comment:
> only been recently added

Yes, on laravel middleware, but this is a third party package with own middlewares, therefore it would be available for any version of laravel
come on, do a test on laravel 8.x

Reply:
If you see https://github.com/laravel/framework/pull/46362, there is no contract, `using` is a helper for avoid this
```php
MyMiddleware::class . ':' .implode('|', ['my_value1', 'my_value1']) . ',' . 'my_argument_2'
```

Reply:
I think you may be confused to which part I added this sentence; it is specifically located as an example for calling the *Laravel* middleware `\Illuminate\Auth\Middleware\Authorize`, it's not about the middlewares this package provides.

Reply:
yeah i was confused
But still, that part of the documentation does not seem necessary, Laravel already comes with the alias `can` by default.
It would be better to keep the documentation as simple as possible, I imagine that Laravel will put that part in its documentation in the near future, so it would be unnecessary to duplicate it here.

Reply:
but leave it, @drbyte has the last word